quickstart.md

```diff
@@ -7,62 +7,28 @@ nav_order: 2
 
-    - Envoy Proxy listens to requests and proxies them to the correct destination
+- Fyde Proxy Orchestrator ensures that Envoy Proxy is configured with the correct requests and takes care of requests authorization and authentication
 
-    - Fyde Proxy Orchestrator ensures that Envoy Proxy is configured with the correct requests and takes care of requests authorization and authentication
+- Fyde Proxy Orchestrator requires a valid token (Fyde Access Proxy enrollment link) that contains the necessary information to bootstrap and authorize the service
 
-    - Fyde Proxy Orchestrator requires a valid token (Fyde Access Proxy enrollment link) that contains the necessary information to bootstrap and authorize the service
-
-1. [Install Fyde Access Proxy in Docker](#install-fyde-access-proxy-in-docker)
-
-    - In this quick start we recomend installing the solution using the docker infrastructure, for other platforms please visit [Fyde Access Proxy - Install steps](fyde_access_proxy.md##install)
-
-    - The public port will need to be available for the clients connecting to resources
-
-1. [Add new resource](#add-new-resource)
-
-    - In this step we are going to create a test resource, however you can use an existing resource of your own instead
-
-1. [Add new user](#add-new-user)
-
-    - In order to register a device, we need to create a user
-
-    - Please note that users and account administrators are separate entities. The same email can be used for an account administrators and a regular user and there will be no relation between them
-
-1. [Enroll a device](#enroll-a-device)
-
-    - A user can have multiple devices, this will be determined by the account administrators
-
-    - In this step we enroll a new device
-
-1. [Access created resource](#access-created-resource)
-
-    - The enrolled device will now access the created resource
-
-1. [Troubleshoot](#troubleshoot)
-
-    - If the device is not accessing the resource we provide these troubleshooting steps
-
-## Add new Fyde Access Proxy
-
-- To add a Fyde Access Proxy, go to **Access** and navigate to **Proxies**. Click the **“+”** icon in the top right.
+- To add a Fyde Access Proxy, go to [Fyde Enterprise Console](http://enterprise.fyde.com){:target="_blank"}, navigate to **Access** and then **Proxies**. Click the **“+”** icon in the top right.
 
 - This proxy will be used to test an access for the test resource, however this doesn't prevent it from being used as a permanent proxy later on
 
     ![Control Screen]({{ site.baseurl }}{% link imgs/ec-access-proxies.png %})
 
 - Add the following:
 
-- **Proxy Name**: Used to identify the proxy
-- **Location**: Optional, for your reference only
-- **Host**: IP/DNS record used to reach the proxy
-- **Port**: Port where the Fyde Access Proxy will be available
+  - **Proxy Name**: Used to identify the proxy
+  - **Location**: Optional, for your reference only
+  - **Host**: IP/DNS record used to reach the proxy
```

Comment:
I would use "host name" instead of IP/DNS

Reply:
Makes sense. Changed.